source/wait_time.c

```diff
@@ -0,0 +1,27 @@
+#ifndef STRUCTS
+#define STRUCTS
+
+#include "../headers/structs.h"
+
+#endif
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+
+
+/*Checking if passengers can finish*/
+void wait_time(transition_system *t) {
+    int i, j , p_W;
+    passenger p;
+    for(i = 0; i < t->length; i++) {
+        p = t->passengers[i];
+        p_W = p.wait_time;
+        for (j = 0; j < t->passengers[i].wait_time ; j++){
```

Comment:
Som beskrevet i hoved-kommentaren gør dette for-loop, at jeres kode ikke overholder kravene fra transition rule 4.13

```suggestion
```